app/components/commitfunds/CommitKnownUserForm.tsx

```diff
@@ -104,6 +105,7 @@ const CommitKnownUserForm = ({
   estimationCoefficient,
 }: ICommitKnownUserFormProps) => {
   let neuAmount: number = estimateNeufromEth(estimationCoefficient)(parseStrToNumStrict(ethAmount));
+  neuAmount = lodash.round(neuAmount, 3);
```

Comment:
if we don't need a number at this point but string we can to `neuAmount.toFixed(3)`. btw. in future don't use let in such cases - just introduce new variable with meaningful name for example: `const neuAmountRounded = ....`

Reply:
`lodash.round` and .toString() is better in this way that `toFixed(3)` returns arbitrary number of decimal places so you can get unnecessary zeroes`1.200`